### PR TITLE
[improve][broker] Add URL check when add/update cluster.

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Currently, we do not do any URL checking when creating/updating clusters via the admin API. 

### Modifications

- Add URL format check when creating/updating clusters.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation
- [x] `doc-not-needed` 
(Please explain why)

### Matching PR in forked repository

PR in forked repository:

### CODE REVIEW

File: pulsar-common/src/main/java/org/apache/pulsar/common/net/ServiceURI.java [hunk 1]
```diff
@@ -106,7 +106,14 @@ public static ServiceURI create(String uriStr) {
 
         return create(uri);
     }
-
+    public static void validate(String urlStr) {
```
Comment (mattisonchao) [inline pulsar-common/src/main/java/org/apache/pulsar/common/net/ServiceURI.java:109]: I'm not sure if we have to enhance this method to check pulsar supported service name.
- pulsar
- pulsar+ssl
- http
- https

Comment (codelipenghui) [reply pulsar-common/src/main/java/org/apache/pulsar/common/net/ServiceURI.java:109]: I think we should

The service URL can only start with pulsar or pulsar+ssl
The web service URL can only start with http or https

Comment (mattisonchao) [reply pulsar-common/src/main/java/org/apache/pulsar/common/net/ServiceURI.java:109]: I see

**Comment by codelipenghui** [comment]
@mattisonchao Please check the failed test.

**Comment by codelipenghui** [comment]
@mattisonchao Please check the failed test.

**Comment by mattisonchao** [comment]
Open another PR to do that.

**Pull request closed without merging**